src/main/java/lihua/main/Main.java

```diff
@@ -0,0 +1,33 @@
+package lihua.main;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import lihua.ui.MainWindow;
+
+/**
+ * A GUI for Duke using FXML.
```

Comment:
should this be "A GUI for Lihua..." to be consistent with the name you decided to name your bot